src/box/vy_log.c

```diff
@@ -1065,7 +1110,12 @@ vy_log_begin_recovery(const struct vclock *vclock, bool force_recovery)
 		 * So in case we are anyway in force recovery mode, let's
 		 * try to delete last .vylog file and continue recovery process.
 		 */
-		if (! force_recovery) {
+		bool is_vylog_empty = vy_log_last_entry_is_snapshot();
+		if (! is_vylog_empty) {
```

Comment:
In the new code we don't use whitespaces after unary operators.

Reply:
Fixed